CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # Changelog
 
+## Version 0.19.0
+
+### Summary
+
+With this release Mitiq supports most recent versions of Python: `3.8`, `3.9` and `3.10`!
+We drop support for Python `3.7`.
+
+Mitiq is now compatible with Numpy `1.21.6`. Different versions of NumPy may not work properly.
+
+Another important update is the addition of new [tools](https://mitiq.readthedocs.io/en/latest/apidoc.html#module-mitiq.pec.representations.biased_noise) for applying learning-based PEC!
+
+Special thanks to the external contributors @yitchen-tim, @amirebrahimi and @isaac-gs!
+
+
+### All Changes
+- Implement conversions of IonQ native gates for Braket (#1525) [@yitchen-tim]
+- [temporary] ignore errors from first-interaction (#1532) [@natestemen]
+- Update sphinx-autodoc-typehints requirement from ~=1.12.0 to ~=1.19.4 (#1515) [@dependabot[bot]]
+- Update pytest-cov requirement from ~=3.0.0 to ~=4.0.0 (#1521) [@dependabot[bot]]
+- Update myst-nb requirement from ~=0.16.0 to ~=0.17.1 (#1529) [@dependabot[bot]]
+- replace deprecated `execution_excludepatterns` (#1530) [@natestemen]
+- Support python 3.10 (#1504) [@natestemen]
+- Fix outdated link in README (2021 -> 2022) (#1528) [@andreamari]
+- Add learning-based PEC subsection in API doc (#1524) [@nathanshammah]
+- remove `print` calls (#1519) [@natestemen]
+- BQSKit tutorial (#1489) [@natestemen]
+- Update pyquil requirement from ~=3.3.0 to ~=3.3.1 (#1516) [@dependabot[bot]]
+- Update scipy requirement from ~=1.7.3 to ~=1.9.1 (#1467) [@dependabot[bot]]
+- Update mypy requirement from ~=0.971 to ~=0.981 (#1511) [@dependabot[bot]]
+- (partially) upgrade numpy (#1501) [@natestemen]
+- Update amazon-braket-sdk requirement from ~=1.30.2 to ~=1.31.0 (#1512) [@dependabot[bot]]
+- Add whitepaper publication info/links (#1456) [@natestemen]
+- Support python 3.9 (#1505) [@natestemen]
+- Learning function for biased noise (#1358) [@misty-w]
```

Comment:
```suggestion
- Learning function for biased noise (#1358) [@Misty-W]
```

Reply:
Oh, sorry for this. I copied it from an old changelog, where I probably did the same mistake :disappointed: